src/poliastro/core/fixed.py

```diff
@@ -0,0 +1,368 @@
+        - 0.00112309 * np.sin(M2)
+        - 0.00011040 * np.sin(M3)
+        - 0.00002539 * np.sin(M4)
+        - 0.00000571 * np.sin(M5)
+    )
+
+    return ra, dec, W
+
+
+@jit
+def venus_rot_elements_at_epoch(T, d):
+    """Calculate rotational elements for Venus.
+
+    Parameters
+    ----------
+    T: float
+        Interval from the standard epoch, in Julian centuries.
+    d: float
+        Interval in days from the standard epoch.
+
+    Returns
+    -------
+    ra, dec, W: tuple (float)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    ra = 272.76
+    dec = 67.16
+    W = 160.20 - 1.4813688 * d
+
+    return ra, dec, W
+
+
+@jit
+def mars_rot_elements_at_epoch(T, d):
+    """Calculate rotational elements for Mars.
+
+    Parameters
+    ----------
+    T: float
+        Interval from the standard epoch, in Julian centuries
+    d: float
+        Interval in days from the standard epoch
+
+    Returns
+    -------
+    ra, dec, W: tuple (float)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    M1 = 198.991226 + 19139.4819985 * T
+    M2 = 226.292679 + 38280.8511281 * T
+    M3 = 249.663391 + 57420.7251593 * T
+    M4 = 266.183510 + 76560.6367950 * T
+    M5 = 79.398797 + 0.5042615 * T
+
+    ra = (
+        317.269202
+        - 0.10927547 * T
+        + 0.000068 * np.sin(M1)
+        + 0.000238 * np.sin(M2)
+        + 0.000052 * np.sin(M3)
+        + 0.000009 * np.sin(M4)
+        + 0.419057 * np.sin(M5)
+    )
+
+    K1 = 122.433576 + 19139.9407476 * T
+    K2 = 43.058401 + 38280.8753272 * T
+    K3 = 57.663379 + 57420.7517205 * T
+    K4 = 79.476401 + 76560.6495004 * T
+    K5 = 166.325722 + 0.5042615 * T
+
+    dec = (
+        54.432516
+        - 0.05827105 * T
+        + 0.000051 * np.cos(K1)
+        + 0.000141 * np.cos(K2)
+        + 0.000031 * np.cos(K3)
+        + 0.000005 * np.cos(K4)
+        + 1.591274 * np.cos(K5)
+    )
+
+    J1 = 129.071773 + 19140.0328244 * T
+    J2 = 36.352167 + 38281.0473591 * T
+    J3 = 56.668646 + 57420.9295360 * T
+    J4 = 67.364003 + 76560.2552215 * T
+    J5 = 104.792680 + 95700.4387578 * T
+    J6 = 95.391654 + 0.5042615 * T
+
+    W = (
+        176.049863
+        + 350.891982443297 * d
+        + 0.000145 * np.sin(J1)
+        + 0.000157 * np.sin(J2)
+        + 0.000040 * np.sin(J3)
+        + 0.000001 * np.sin(J4)
+        + 0.000001 * np.sin(J5)
+        + 0.584542 * np.sin(J6)
+    )
+
+    return ra, dec, W
+
+
+@jit
+def jupiter_rot_elements_at_epoch(T, d):
+    """Calculate rotational elements for Jupiter.
+
+    Parameters
+    ----------
+    T: float
+        Interval from the standard epoch, in Julian centuries
+    d: float
+        Interval in days from the standard epoch
+
+    Returns
+    -------
+    ra, dec, W: tuple (float)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    Ja = 99.360714 + 4850.4046 * T
```

Comment:
This is being computed in degrees and passed in these units to the trigonometric functions below! As pointed out by @Yash-10, the tests within poliastro did not covered all this logic but the validation ones did. Pinging @astrojuanlu, we might discuss this on this week's poliastro meeting (July the 5th, 2021).

Reply:
The same applies to the rest of the bodies, though this was only noticed for the cases of Jupiter and Moon due to the big amount of terms when computing their rotational elements at epoch.

Reply:
Thanks @jorgepiloto ! I will soon open a pull request to (hopefully) rectify my mistake!

Reply:
Yeah, in general I think having the validation tests is awesome, but also we should have more unit tests :)